### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by at-dinhnguyen**

### CODE REVIEW

File: database/migrations/2017_03_13_084443_create_article_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+      Schema::create('article', function (Blueprint $table) {
+          $table->increments('id');
+          $table->string('title');
+          $table->string('summary');
+          $table->string('content');
```
Comment (at-vienle) [inline database/migrations/2017_03_13_084443_create_article_table.php]: `varchar` is not enough for the `content`. Please change.

File: database/migrations/2017_03_13_084443_create_article_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+class CreateArticleTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+      Schema::create('article', function (Blueprint $table) {
```
Comment (at-vienle) [inline database/migrations/2017_03_13_084443_create_article_table.php]: the table name should use plural noun, not single noun.

Comment (at-vienle) [inline database/migrations/2017_03_13_084443_create_article_table.php]: please add column to know who create a article.

File: database/migrations/2017_03_13_084443_create_article_table.php (hunk 1 above)
Comment (at-dinhnguyen) [inline database/migrations/2017_03_13_084443_create_article_table.php]: Ok, I will change it as belows:
$table->text('description');	
TEXT equivalent for the database.

File: database/migrations/2017_03_13_084443_create_article_table.php (hunk 2 above)
Comment (at-dinhnguyen) [inline database/migrations/2017_03_13_084443_create_article_table.php]: Oh, I will rename it from Article to Articles

Comment (at-dinhnguyen) [inline database/migrations/2017_03_13_084443_create_article_table.php]: I have been add new column: author_name ! 
I think it not suitable if we need to expand project ! 
But i will solve it as you comment !

File: database/migrations/2017_03_16_075405_add_votes_to_Articles_table.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+class AddVotesToArticlesTable extends Migration
+{
+  /**
+   * Run the migrations.
+   *
+   * @return void
+   */
+  public function up()
+  {
+    Schema::create('articles', function (Blueprint $table) {
```
Comment (at-vienle) [inline database/migrations/2017_03_16_075405_add_votes_to_Articles_table.php]: Please fix on the file that I reviewed, DON'T create new migration file. Now if you clear db and run migration then on DB there are 2 table `article` and `articles`. Do you think it is ok?

File: database/migrations/2017_03_16_075405_add_votes_to_Articles_table.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        $table->increments('id');
+        $table->string('title');
+        $table->string('author_name');
```
Comment (at-vienle) [inline database/migrations/2017_03_16_075405_add_votes_to_Articles_table.php]: I don't mean create a text column, I want to add a foreign key. What happen if user update the name but on this table still keep the old name.

**Pull request merged**